atomic_reactor/inner.py

```diff
@@ -603,9 +603,6 @@ class ImageBuildWorkflowData(ISerializer):
     built_image_inspect: Dict[str, Any] = None
     layer_sizes: List[Dict[str, Union[str, int]]] = field(default_factory=list)
 
-    # list of images pulled during the build, to be deleted after the build
-    pulled_base_images: Set = field(default_factory=set)
```

Comment:
IIRC there was special handling for `set` objects somewhere in this class, can we also remove that now?